apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.html

```diff
@@ -44,33 +50,61 @@ <h2>Accordion with list</h2>
     list-styling to ensure that the list looks correct without further adjustments.
   </p>
   <cookbook-example-viewer [html]="accordionWithListExample.template">
-    <cookbook-accordion-with-list-example
-      #accordionWithListExample
-    ></cookbook-accordion-with-list-example>
+    <div class="example-frame">
+      <cookbook-accordion-with-list-example
+        #accordionWithListExample
+      ></cookbook-accordion-with-list-example>
+    </div>
   </cookbook-example-viewer>
 
   <h3>Additional styling</h3>
   <p>
-    Similarly to the simple accordion, the accordion with a list can also be wrapped in a Kirby Card to
-    make it stand out from the canvas.
+    Similarly to the simple accordion, the accordion with a list can also be wrapped in a Kirby Card
+    to make it stand out from the canvas.
   </p>
   <cookbook-example-viewer [html]="accordionWithListInCardExample.template">
-    <cookbook-accordion-with-list-in-card-example
-      #accordionWithListInCardExample
-    ></cookbook-accordion-with-list-in-card-example>
+    <div class="example-frame">
+      <cookbook-accordion-with-list-in-card-example
+        #accordionWithListInCardExample
+      ></cookbook-accordion-with-list-in-card-example>
+    </div>
   </cookbook-example-viewer>
 
   <h2>Accordion toggle event</h2>
-  <p>
-    A toggle event is emitted whenever an accordion is expanded or collapsed.
-  </p>
+  <p>A toggle event is emitted whenever an accordion is expanded or collapsed.</p>
   <cookbook-example-viewer
     [html]="accordionEventExample.template"
     [ts]="accordionEventExample.codeSnippet"
   >
-    <cookbook-accordion-events-example #accordionEventExample></cookbook-accordion-events-example>
+    <div class="example-frame">
+      <cookbook-accordion-events-example #accordionEventExample></cookbook-accordion-events-example>
+    </div>
   </cookbook-example-viewer>
 
+  <h2>Accessibility</h2>
+  <p>
+    To enhance accessibility and improve screen reader navigation, a
+    <code>headingLevel</code>
+    can be assigned to the accordion headers to include them in the pages heading structure.
+    Headings use the
+
+    <!-- prettier-ignore -->
+    <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/heading_role">heading</a>
+    role.
+  </p>
+  <p>
+    The
+    <code>aria-level</code>
+    attribute is a required part of the heading role and indicates to assistive technologies that
+    the element should be treated as a heading. For example, applying
+    <code>headingLevel="1"</code>
+    to the accordion defines the header as the main heading of the page. A level 2 heading would
+    represent the first subsection, a level 3 a subsection of that, and so on. Heading levels can be
```

Comment:
Really nice! I think we should leave out the technical details about the attributes we add under the hood, as it seems less relevant for readers and might confuse.

```suggestion
Heading levels can be
```

Reply:
Now that I see it, I think it would be good to add a new example under the _Accessibility_ header, that shows the headingLevel example, instead of sort of hiding it away in the first example.